### DISCUSSION TITLE
Displacement of tower fore-aft bending mode

### DISCUSSION THREAD

**Discussion by gabriellelarzilliere** [OpenFAST/openfast#2036] [Q&A]
I have a doubt about the differences between the "Displacement of 1st tower fore-aft bending mode DOF" and the "Displacement of 2nd tower fore-aft bending mode DOF". What is each of them representing in my openfast simulation?

**Comment by jjonkman** [comment]
Dear @gabriellelarzilliere,

The tower-bending degrees of freedom in ElastoDyn are factors that scale the user-specified mode shapes (shape functions, whose value at the tower top is unity) at the top of the tower. That is, the actual transverse tower displacements are linear combinations of the degree of freedom multiplied by the shape functions.

Best regards,

**Reply by gabriellelarzilliere** [reply]
But so is there a difference between the fact that one is 1st tower and the other second tower. Do they correspond to different modes of deflection for example?

**Reply by jjonkman** [reply]
Dear @gabriellelarzilliere,

Yes, each tower DOF in ElastoDyn corresponds to a different bending mode shape (the 1st and 2nd modes, respectively).  See Figure 10 from the old FAST v6 User's Guide for an illustration: https://openfast.readthedocs.io/en/main/_downloads/d8bd014121d6505cb25cf49bee5eaa80/Old_FAST6_UsersGuide.pdf.

Best regards,
